### ISSUE TITLE
LocalStack: VPC end-point API is not working

### ISSUE THREAD

**Issue by RSHUKL01** [localstack/localstack#2650]
LocalStack: VPC end-point API is not working

Command Used: 
aws ec2 describe-vpc-endpoint-services --endpoint-url=http://localhost:443

Actual Error:
Unable to parse response (syntax error: line 1, column 54), invalid XML received. Further retries may succeed:
b'<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">\n<title>500 Internal Server Error</title>\n<h1>Internal Server Error</h1>\n<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>\n'

Docker Log:
 NotImplementedError: The describe_vpc_endpoint_services action has not been implemented

**Comment by whummer** [comment]
Thanks for reporting @RSHUKL01 . This has been fixed upstream in spulec/moto#3108 and should now be available in the `latest` version of the Docker image. Please report here if the problem persists. Thanks

**Comment by jkbuster** [comment]
This still appears to be an issue.  Just did a fresh image pull, and I get this:


```
# aws ec2 describe-vpc-endpoint-services

Unable to parse response (syntax error: line 1, column 54), invalid XML received. Further retries may succeed:
b'<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">\n<title>500 Internal Server Error</title>\n<h1>Internal Server Error</h1>\n<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>\n'
```

Same with the `describe-vpc-endpoints` command.
